src/modules/tsilo/ts_append.h

```diff
@@ -22,7 +22,7 @@
 #ifndef _TS_APPEND_H
 #define _TS_APPEND_H
 
-int ts_append(struct sip_msg* msg, str *ruri, char *table);
-int ts_append_to(struct sip_msg* msg, int tindex, int tlabel, char *table, str *uri);
+int ts_append(struct sip_msg* msg, str *ruri, str *contact, char *table);
```

Comment:
Any specific reason to have the contact arg inserted between ruri and table args, while in ts_append_to is put at the end?

Reply:
To be precise, the only reason was to keep it just after the ruri parameter.
So that in both cases with ts_append() and ts_append_to() I have put it unconsciously just after uri (ruri).